spec/connect/hwinfo/s390_spec.rb

```diff
@@ -15,30 +15,7 @@
 
   after(:each) do
     SUSE::Connect::HwInfo::Base.instance_variable_set('@arch', nil)
-  end
-
-  it 'returns a hwinfo hash for x86/x86_64 systems' do
-    expect(Open3).to receive(:capture3).with(shared_env_hash, 'read_values -u').and_return(['', '', success])
-
-    hwinfo = subject.hwinfo
-    expect(hwinfo[:hostname]).to eq 'test'
-    expect(hwinfo[:cpus]).to eq 1
-    expect(hwinfo[:sockets]).to eq 1
-    expect(hwinfo[:hypervisor]).to eq 'z/VM 6.1.0'
-    expect(hwinfo[:arch]).to eq 's390x'
-    expect(hwinfo[:uuid]).to eq nil
-  end
-
-  it 'returns system cpus count' do
```

Comment:
Hmm.. If no one uses the `#cpus` or `#sockets` methods other than `#hwinfo`, then we should mark them private and then they do not need to be tested.
But if they are part of the public interface of this class, then I think we need these tests.